scripts/servers.py

```diff
@@ -722,3 +722,42 @@ def managed_webdriverio_server(
             yield proc
     finally:
         del os.environ['MOBILE']
+
+
+@contextlib.contextmanager
+def managed_acceptance_tests_server(
+    suite_name: str = 'full',
+    stdout: int = subprocess.PIPE,
+) -> Iterator[psutil.Process]:
+    """Returns context manager to start/stop the Acceptance server gracefully.
+
+    Args:
+        suite_name: str. The suite name whose tests should be run. If the value
+            is `full`, all tests will run.
+        stdout: int. This parameter specifies the executed program's standard
+            output file handle.
+
+    Yields:
+        psutil.Process. The webdriverio process.
+
+    Raises:
+        ValueError. Number of sharding instances are less than 0.
+    """
+    acceptance_tests_args = [
+        common.NODEMODULES_JASMINE_BIN_PATH,
+        '--config="%s"' % common.JASMINE_CONFIG_FILE_PATH,
+        '--filter="%s"' % suite_name
+    ]
+
+    # OK to use shell=True here because we are passing string literals and
+    # constants, so there is no risk of a shell-injection attack.
```

Comment:
Is this true? `suite_name` is not a constant

Reply:
`suite_name` will be a constant . SInce all that logic is behind the CI workflow. No intermediatery can change the constant value of `suite_name` in between. But i'll look again to make sure it is safe